evap/staff/importers.py

```diff
@@ -427,13 +500,11 @@ def check_evaluation_data_correctness(self, semester):
         missing_degree_names = set()
         missing_course_type_names = set()
         for evaluation_data in self.evaluations.values():
-            if Course.objects.filter(semester=semester, name_en=evaluation_data.name_en).exists():
-                self.errors[ImporterError.COURSE].append(
-                    _("Course {} does already exist in this semester.").format(evaluation_data.name_en)
-                )
-            if Course.objects.filter(semester=semester, name_de=evaluation_data.name_de).exists():
-                self.errors[ImporterError.COURSE].append(
-                    _("Course {} does already exist in this semester.").format(evaluation_data.name_de)
+            if evaluation_data.check_existing_course(semester, self):
+                self.warnings[ImporterWarning.DUPL].append(
```

Comment:
I'd say we should use a new warning category for this -- it's not really a duplicate.

Reply:
What do you think of

- `MODIF = ("modification", gettext_lazy("Possible modifications"), x)`
- `EXISTS= ("existing", gettext_lazy("Existing courses", x)`
- `ADJ= ("adjustments", gettext_lazy("Automatic adjustments", x)`

Or any combination of these?

Reply:
One could see the merging of degrees and adding the user to the evaluation as modification/adjustment of the course/evaluation. At least that is where the thought came from. But maybe `EXISTS` is the best out of these...

Reply:
i'd say `EXISTS` suffices and you can put all of these into there